maquetado/scss/navbar.scss

```diff
@@ -0,0 +1,36 @@
+.navbar {
+  background-color: $white;
+  box-shadow: 0 4px 10px $black-russian;
+  display: flex;
+  flex-direction: column;
+  max-height: 70px;
+  max-width: 1366px;
+  position: fixed;
+  width: 1366px;
```

Comment:
Change to width: 100% because you set the width that you want with the max-width